test/Microsoft.AspNetCore.Mvc.FunctionalTests/BasicTests.cs

```diff
@@ -50,7 +50,7 @@ public BasicTests(MvcTestFixture<BasicWebSite.Startup> fixture)
 #if GENERATE_BASELINES
             ResourceFile.UpdateFile(_resourcesAssembly, outputFile, expectedContent, responseContent);
 #else
-            Assert.Equal(expectedContent, responseContent, ignoreLineEndingDifferences: true);
+            Assert.Equal(expectedContent.Trim(), responseContent, ignoreLineEndingDifferences: true);
```

Comment:
For a little while, some developers weren't aware of `ignoreLineEndingDifferences`. Have a feeling something at a lower level is going South here due to line ending expectations.

Bottom line, we should figure out why `ignoreLineEndingDifferences` is insufficient rather than add `.Trim()`. That's not on you. But, please file a bug describing what tests fail after
1. `git clone --config core.autocrlf false https://github.com/aspnet/Mvc.git`
2. `cd Mvc`
3. `.\build.cmd`